rest/route/annotations.go

```diff
@@ -203,6 +203,81 @@ func (h *annotationByTaskGetHandler) Run(ctx context.Context) gimlet.Responder {
 	return gimlet.NewJSONResponse(res)
 }
 
+////////////////////////////////////////////////////////////////////////
+//
+// PATCH /rest/v2/tasks/annotations
+
+type bulkCreateAnnotationsOpts struct {
+	TaskUpdates []annotations.TaskUpdate `bson:"task_updates" json:"task_updates"`
+}
+
+type bulkPatchAnnotationHandler struct {
+	user gimlet.User
+	opts bulkCreateAnnotationsOpts
+}
+
+func makeBulkPatchAnnotations() gimlet.RouteHandler {
+	return &bulkPatchAnnotationHandler{}
+}
+
+func (h *bulkPatchAnnotationHandler) Factory() gimlet.RouteHandler {
+	return &bulkPatchAnnotationHandler{}
+}
+
+func (h *bulkPatchAnnotationHandler) Parse(ctx context.Context, r *http.Request) error {
+	var err error
+	body := utility.NewRequestReader(r)
+	defer body.Close()
+	err = json.NewDecoder(body).Decode(&h.opts)
+	if err != nil {
+		return gimlet.ErrorResponse{
+			Message:    fmt.Sprintf("API error while unmarshalling JSON: '%s'", err.Error()),
```

Comment:
@Kimchelly should this be `unmarshalling JSON: '%s'`, or should it be `API error while unmarshalling JSON: '%s'` since it's specifically an API error?

Reply:
This can be simplified down to just `return errors.Wrapf(err "unmarshalling options from JSON request body")` since [`Parse` already returns Bad Request in the Gimlet framework when the error is not of type `gimlet.ErrorResponse`](https://github.com/evergreen-ci/gimlet/blob/b882e01673e7fb37badc0d8f04cac8e308d2f9ec/framework.go#L43-L46).

This error message probably doesn't matter since I'm going to fix all the error messages in this PR after I rebase.